### DISCUSSION TITLE
Vue.js not detected in VuePress 2.

### DISCUSSION THREAD

**Discussion by An-n-ya** [vuepress/core#342] [Q&A] [answered]
I thought is was the bug of my vue-devtool extension. But I found that the vue-devtool works on VuePress 1 project. I also found that the due-devtool even cannot work on the homepage of VuePress 2 (https://v2.vuepress.vuejs.org/). Is it a bug of VuePress2?

**Comment by Mister-Hope** [comment, marked as answer]
Edited: VuePress has added support for Vue DevTools, and you must use DevTools@v6
